homeassistant/components/unifi/device_tracker.py

```diff
@@ -431,7 +431,12 @@ def extra_state_attributes(self):
         if self.device.state == 0:
             return {}
 
-        attributes = {}
+        attributes = {
+            CONNECTION_NETWORK_MAC: self.device.mac,
```

Comment:
Mac address should be exposed in the corresponding property.

https://developers.home-assistant.io/docs/core/entity/device-tracker#properties